crates/rslint_parser/src/ast/stmt_ext.rs

```diff
@@ -253,13 +253,24 @@ impl VarDecl {
     }
 }
 
-impl ImportDecl {
+pub trait Source {
```

Comment:
I think it's better to add those methods on the structs instead of introducing a new trait.